packages/@aws-cdk/aws-backup/lib/rule.ts

```diff
@@ -58,6 +58,13 @@ export interface BackupPlanRuleProps {
    * common vault for the plan will be created
    */
   readonly backupVault?: IBackupVault;
+
+  /**
+   * Enables continuous backup and point-in-time restores (PITR).
```

Comment:
Doc should mention requirement that `deleteAfter` is set and `moveToColdStorageAfter` is not set.